packages/ckeditor5-horizontal-line/src/horizontallinecommand.ts

```diff
@@ -68,7 +67,7 @@ function isHorizontalLineAllowedInParent( selection: Selection | DocumentSelecti
  * @param model Model instance.
  */
 function getInsertHorizontalLineParent( selection: Selection | DocumentSelection, model: Model ): Element {
-	const insertionRange = findOptimalInsertionRange( selection, model );
+	const insertionRange = model.schema.findOptimalInsertionRange( selection, model );
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa